rotkehlchen/chain/ethereum/modules/l2/zksync.py

```diff
@@ -0,0 +1,589 @@
+        for new_transactions in self._query_zksync_api_transactions(
+                address=address,
+                from_hash=from_hash,
+                direction=direction,
+        ):
+            if len(new_transactions) == 0:
+                continue
+
+            unique_transactions = set(new_transactions)
+            existing_txs = len(unique_transactions.intersection(input_transactions))
+            if existing_txs != 0:
+                log.debug(f'Got {existing_txs} already queried transactions during pagination')
+                unique_transactions -= input_transactions
+
+            with self.database.conn.write_ctx() as write_cursor:
+                self._add_zksynctxs_db(
+                    write_cursor=write_cursor,
+                    transactions=unique_transactions,
+                )
+                if direction == 'older':
+                    current_start_ts = new_transactions[-1].timestamp
+                else:  # direction -> newer
+                    current_end_ts = new_transactions[-1].timestamp
+                ranges.update_used_query_range(
+                    write_cursor=write_cursor,
+                    location_string=location,
+                    queried_ranges=[(current_start_ts, current_end_ts)],
+                )
+
+            input_transactions |= unique_transactions
+
+    def _create_tokens_mapping(self) -> None:
+        from_idx = 0
+        while True:
+            options = {'from': from_idx, 'direction': 'newer', 'limit': ZKSYNCLITE_MAX_LIMIT}
+            response = self._query_api(url='tokens', options=options)
+            result = response.get('list', None)
+            if result is None:
+                msg = 'Unexpected zksync lite tokens response. Could not find result in the response.'  # noqa: E501
+                log.error(f'{msg} Response: {response}')
+                raise RemoteError(msg)
+
+            asset: CryptoAsset
+            for entry in result[0 if from_idx == 0 else 1:]:
+                try:  # if not first page we skip first since the from_idx repeats first entry
+                    token_id = entry['id']
+                    token_symbol = entry['symbol']
+                    address = deserialize_evm_address(entry['address'])
+                    if address == ZERO_ADDRESS:
+                        asset = self.eth
+                    else:
+                        try:
+                            asset = get_or_create_evm_token(
+                                userdb=self.database,
+                                evm_address=address,
+                                chain_id=ChainID.ETHEREUM,
+                                evm_inquirer=self.ethereum.node_inquirer,
+                                encounter=TokenEncounterInfo(description='Querying zksync tokens mapping'),  # noqa: E501
+                            )
+                        except UnknownAsset:
+                            log.warning(
```

Comment:
I believe it is `NotERC20Conformant` what you want here 

```suggestion
                        except NotERC20Conformant:
                            log.warning(
```